theano/gpuarray/tests/check_dnn.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
```

Comment:
I think in the end, we may want a Python script, as it would be able to interact more easily with the "normal" test cases, for instance inheriting from them, etc.